examples/burial-form/ConfirmationPage.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Page} from '@department-of-veterans-affairs/va-forms-system-core';
+
+export default function ConfirmationPage(props) {
+  return (
+    <>
+      <Page {...props}>
```

Comment:
Why do we need to use a `Page` component here?

Reply:
This is simply a placeholder for the ConfirmationPage component until someone actually picks up that card, as I needed it for testing purposes. I will add a note to that task letting them know that the initial scaffolding is there but the UI needs to be finished.